### DISCUSSION TITLE
Can't select YT installation file from ReVanced Media File Picker on HyperOS

### DISCUSSION THREAD

**Discussion by DantesNoWar** [ReVanced/revanced-discussions#1110] [Question]
Hi, I've uninstalled Revanced because many errors were occuring recently.
Then I've tried to reinstall it and repatch it.
The problem is that I'm not able to select any file via the File Picker. All YT files are greyed out. 

I'm using a Xiaomi HyperOS phone, and a couple of days ago I've debloated my phone with Canta and Shizuku.
The problem is that I can't figure out what makes the files grey during the file selection inside ReVanced (the 
![Screenshot_2024-12-04-01-42-54-602_com google android documentsui](https://github.com/user-attachments/assets/6b3f23a0-d8d3-4909-a5bb-3d7b4d1cec8e)
![Stitch19390aa2c81](https://github.com/user-attachments/assets/d1b6aa21-7d2c-443b-91b2-942fc1d9f8cb)
latest version available this week). 

Can someone help me solve this?
I'll attach a screenshot with the issue and a longshot with all the apps I've uninstalled (mainly recommended and safe to remove apps).

Let me know if you need more details.
I post this because I didn't found a similar problem and nobody helped me elsewhere.

**Comment by oSumAtrIX** [comment]
You can only select APK files 
![image](https://github.com/user-attachments/assets/18a46de8-6c31-4618-8bf9-79cc651c716e)

**Reply by DantesNoWar** [reply]
![Screenshot_2024-12-05-23-21-51-106_com brave browser_beta](https://github.com/user-attachments/assets/64e10bdc-365a-47c8-913f-4433bfa477bb)

I downloaded this file from [ApkMirror](https://www.apkmirror.com/apk/google-inc/youtube/youtube-19-46-42-release/youtube-19-46-42-android-apk-download/download/?key=f5542151a2b5f18b6e8037a6364d338070f10be6) and it an ".apkm". I just tried to get rid of the m from the name and now it was selectable.
But when selected, the screen come back to the ReVanced Patch selector without doing anything. I tried three times and it keeps doing that. 
Something's off, and I have a bad feeling about some apps I've uninstalled.
I've read all th e description before uninstalling them and everything seemed safe. Maybe not, or maybe is something else?

**Reply by oSumAtrIX** [reply]
You need an APK not an APKM as an APK

**Reply by DantesNoWar** [reply]
I've solved this downloading from ApkPure. You were right, thanks a lot! 🙏
